docs/cloud.md

```diff
@@ -0,0 +1,242 @@
+# Feldera Cloud
+
+These instructions explain how to run the Feldera Platform on a AWS Elastic Kubernetes Service (EKS). 
+
```

Comment:
I'dd add some note here that if someone wants to use/try cloud to contact us (and that it's not in the feldera repo)

Reply:
`learnmore@feldera.com`

Reply:
Take a look at the last commit.